### PR TITLE
Bug 1966675: Update Image Stream to ImageStream

### PR DESCRIPTION
**Pull request by kdoberst**
This addresses [Bug 1966675](https://bugzilla.redhat.com/show_bug.cgi?id=1966675)

In translation files (aka public.json), changed used references from "Image Stream" to "ImageStream".

### CODE REVIEW

File: frontend/public/components/image-stream.tsx [hunk 1]
```diff
@@ -167,17 +167,17 @@ export const ExampleDockerCommandPopover: React.FC<ImageStreamManipulationHelpPr
         <div>
           <p>
             {t(
-              'public~Create a new Image Stream Tag by pushing an image to this Image Stream with the desired tag.',
+              'public~Create a new ImageStream Tag by pushing an image to this ImageStream with the desired tag.',
```
Comment (rebeccaalpert) [inline frontend/public/components/image-stream.tsx:170]: The first one should probably be ImageStreamTag; it looks like that's a specific resource kind: https://github.com/openshift/console/blob/master/frontend/public/models/index.ts#L326

File: frontend/public/locales/ko/public.json [hunk 2]
```diff
@@ -489,13 +489,13 @@
   "Unresolved": "미해결",
   "There is no image associated with this tag": "이 태그와 관련된 이미지가 없습니다.",
   "Image registry commands": "이미지 레지스트리 명령",
-  "Create a new Image Stream Tag by pushing an image to this Image Stream with the desired tag.": "원하는 태그를 사용하여 이미지를이 이미지 스트림에 푸시하여 새 이미지 스트림 태그를 만듭니다.",
+  "Create a new ImageStream Tag by pushing an image to this ImageStream with the desired tag.": "원하는 태그를 사용하여 이미지를이 이미지 스트림에 푸시하여 새 이미지 스트림 태그를 만듭니다.",
```
Comment (rebeccaalpert) [inline frontend/public/locales/ko/public.json:492]: We're going to do one more file exchange with the translation team before code freeze, so there's no need to manually update the files in ko, zh, and ja.

**Review by rebeccaalpert — CHANGES_REQUESTED** [review]
Hi Kim! This looks great; I just had a couple of comments.

File: frontend/public/components/image-stream.tsx (hunk 1 above)
Comment (kdoberst) [reply frontend/public/components/image-stream.tsx]: Made change to ImageStreamTag.

**Review by rebeccaalpert — APPROVED** [review]
Thank you so much! Looks great.

File: frontend/public/locales/ko/public.json (hunk 2 above)
Comment (kdoberst) [reply frontend/public/locales/ko/public.json]: I removed the changes to the ko, zh, and ja public.json files

**Review by spadgett — APPROVED** [review]
/approve

**Review by zherman0 — APPROVED** [review]

**Pull request merged**